packages/server/src/main/java/software/uncharted/terarium/hmiserver/controller/knowledge/KnowledgeController.java

```diff
@@ -797,14 +798,23 @@ public ResponseEntity<Void> variableExtractions(
 	)
 	public ResponseEntity<Void> pdfExtractions(
 		@RequestParam("document-id") final UUID documentId,
-		@RequestParam(name = "domain", defaultValue = "epi") final String domain,
-		@RequestParam(name = "project-id", required = false) final UUID projectId
+		@RequestParam(name = "project-id", required = false) final UUID projectId,
+		@RequestParam(name = "wait", required = false, defaultValue = "false") final Boolean wait
```

Comment:
Could we use the same ASYNC/SYNC nomenclature used for taskrunner?

Reply:
seconded!

Reply:
Sure